apps/server/src/service/dexLiquidityPoolService.ts

```diff
@@ -62,40 +62,41 @@ export class DexLiquidityPoolService {
 
   async getBatchOrders(lock: Script): Promise<OrderHistory[]> {
     const liquidityOrders: OrderHistory[] = [];
-    const orderLock1 = ScriptBuilder.buildLiquidityOrderLockScript();
-    const removeQueryOptions1: QueryOptions = {
-      lock: {
-        script: orderLock1.toLumosScript(),
-        argsLen: 'any',
-      },
-      order: 'desc',
-      fromBlock: BLOCK_NUMBER,
-    };
-
-    const orderLock2 = ScriptBuilder.buildSudtSudtLiquidityOrderLockScript();
-    const removeQueryOptions2: QueryOptions = {
-      lock: {
-        script: orderLock2.toLumosScript(),
-        argsLen: 'any',
-      },
-      order: 'desc',
-      fromBlock: BLOCK_NUMBER,
-    };
 
     const userLockHash = lock.toHash().slice(2, 66);
-    const removeTxs1 = await this.dexRepository.collectTransactions(removeQueryOptions1, true, true);
-    const removeTxs2 = await this.dexRepository.collectTransactions(removeQueryOptions2, true, true);
     const infoCells = await this.getLiquidityPools();
-    infoCells.forEach((x) => {
+    infoCells.forEach(async (x) => {
```

Comment:
Maybe this will cause a bug.,`forEach` is a sync method, but you are trying to modify the `liquidityOrders` in the `forEach`

Reply:
ok, it's fixed